src/webhook/subscription_validation.py

```diff
@@ -0,0 +1,45 @@
+from flask import request, Blueprint
+import sys, traceback
+import json
+import os
+
+from webhook.request_helper import validate, responses
+
+bp = Blueprint("subscription-webhook", __name__)
+
+valid_source_namespaces = os.getenv("SUBSCRIPTION_VALIDATION_NAMESPACES", "openshift-operators")
+
+valid_source_namespaces = valid_source_namespaces.split(",")
+
+@bp.route('/subscription-validation', methods=('GET','POST'))
```

Comment:
Any reason for supporting the GET method as well? Considering the admission controller does a POST

Reply:
Not particularly. (The group one accepts `GET`, too) Happy to 🔪 them.